### PR TITLE
Fix remove redundant import

### PR DESCRIPTION
**Pull request by FinalT**
remove redundant import
#2460

### CODE REVIEW

File: protocol/triple/triple-tool/gen/generator/tripleTpl.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ const ImportTpl = `
 
 import (
 	context "context"
-	http "net/http"
+	"net/http"
```
Comment (DMwangnima) [inline protocol/triple/triple-tool/gen/generator/tripleTpl.go:156]: Add inspection for non-stream scenario.

File: protocol/triple/triple-tool/gen/generator/tripleTpl.go [hunk 2]
```diff
@@ -153,7 +153,7 @@ const ImportTpl = `
 
 import (
```
Comment (DMwangnima) [inline protocol/triple/triple-tool/gen/generator/tripleTpl.go:154]: Please remove other redundant alias when there is no conflict.

File: protocol/triple/triple-tool/gen/generator/genTriple.go [hunk 3]
```diff
@@ -93,6 +93,9 @@ func (g *Generator) parseProtoToTriple(p *proto.Proto) (TripleGo, error) {
 						StreamsReturn:  vi.StreamsReturns,
 					}
 					s.Methods = append(s.Methods, md)
+					if md.StreamsReturn || md.StreamsReturn {
```
Comment (DMwangnima) [inline protocol/triple/triple-tool/gen/generator/genTriple.go:96]: ```if md.StreamsRequest || md.StreamsReturn {```

**Comment by DMwangnima** [comment]
Please run ```go mod tidy``` under dubbo-go directory.

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**